sdk/deviceupdate/azure-mgmt-deviceupdate/sdk_packaging.toml

```diff
@@ -0,0 +1,8 @@
+[packaging]
+package_name = "azure-mgmt-deviceupdate"
+package_nspkg = "azure-mgmt-nspkg"
+package_pprint_name = "Deviceupdate Management"
+package_doc_id = ""
+is_stable = false
+is_arm = true
+need_msrestazure = true
```

Comment:
`need_msrestazure = false`
`need_azuremgmtcore = true`

Do we need `package_nspkg = "azure-mgmt-nspkg"`?